tests/sim_devices/devices.yaml

```diff
@@ -253,6 +253,9 @@ resources:
   TCPIP::MSO2KB-HOSTNAME::INSTR:
     device: mso2kb
     filename: scope/mso2kb.yaml
+  USB0::0x0699::0x03A4::C021204::INSTR:
```

Comment:
Unless this is used in a test, it doesn't need to be added here.

Reply:
I don't fully understand the automated testing, but I had added that in while I tried to figure it out. I've removed it.